e2e/subscription.go

```diff
@@ -306,15 +306,15 @@ func (t *DeployManager) deleteClusterObjects(co *clusterObjects) error {
 }
 
 // UninstallLVM uninstalls lvm operator.
-func (t *DeployManager) UninstallLVM(lvmCatalogImage string, subscriptionChannel string) error {
+func UninstallLVM(lvmCatalogImage string, subscriptionChannel string) error {
```

Comment:
start function with lower case if its not used outside the package. Please check this for all the other functions as well.

Reply:
Updated.